spikeinterface/curation/remove_duplicated_spikes.py

```diff
@@ -0,0 +1,159 @@
+from typing import Optional
+import numpy as np
+from spikeinterface.core import BaseSorting, BaseSortingSegment
+from spikeinterface.core.core_tools import define_function_from_class
+
+try:
+	import numba
```

Comment:
same indentation problem @DradeAW

Reply:
..... why does GitHub understand tabs as equal to 8 spaces !?

Converting to spaces